src/__tests__/components/OutlinedTextInput.test.js

```diff
@@ -0,0 +1,45 @@
+/* globals describe it expect */
+/* eslint-disable flowtype/require-valid-file-annotation */
+
+import * as React from 'react'
+import ShallowRenderer from 'react-test-renderer/shallow'
+
+import { OutlinedTextInput } from '../../components/themed/OutlinedTextInput.js'
+
+describe('OutlinedTextInput', () => {
+  it('should render with some props', () => {
+    const renderer = new ShallowRenderer()
+
+    const props = {
+      value: 'string',
+      error: 'string',
+      label: 'string',
+      clearIcon: true, // Defaults to 'true'
+      marginRem: 0.5 | [0.5], // Defaults to 0.5
+      multiline: true, // Defaults to 'false'
+      searchIcon: true, // Defaults to 'false'
+
+      onBlur: () => undefined,
+      onChangeText: () => undefined,
+      onClear: () => undefined,
+      onFocus: () => undefined,
+
+      // Other React Native TextInput properties:
+      autoCapitalize: 'none' | 'sentences' | 'words' | 'characters', // Defaults to 'sentences'
+      autoCorrect: true, // Defaults to 'true'
+      blurOnSubmit: true, // Defaults to 'true'
+      inputAccessoryViewID: 'string',
+      keyboardType: 'default' | 'number-pad' | 'decimal-pad' | 'numeric' | 'email-address' | 'phone-pad', // Defaults to 'default'
+      maxLength: 11,
+      onSubmitEditing: () => undefined,
+      returnKeyType: 'done' | 'go' | 'next' | 'search' | 'send', // Defaults to 'done'
```

Comment:
This should not have the 'or' sign, it was only part of the type.
You should just pass one of the options.